src/components/api.js

```diff
@@ -9,53 +9,50 @@ class Client {
     this.Rest = restClient;
   }
 
-  getCampaignData(campaignId) {
-    return fetch(this.baseUrl + '/contributions/' + campaignId).then(rsp => {
-      return rsp.json();
-    });
+  async getCampaignData(campaignId) {
```

Comment:
based on pitfall 4 in https://medium.com/@bluepnume/learn-about-promises-before-you-start-using-async-await-eb148164a9c8, you have to handle errors using try catch.